src/common/pf_dcp.c

```diff
@@ -66,6 +66,8 @@
 #define PF_DCP_ID_REQ_FRAME_ID  0xfefe
 #define PF_DCP_ID_RES_FRAME_ID  0xfeff
 
+#define PF_DCP_SAM_TIMEOUT_3_SEC 3000000
```

Comment:
Is this is a define that could be set to something other than 3s (could it be shorter maybe)? If so change the name so that it doesn't contain the value.

Reply:
Maybe:
```
/* Client hold time, 3 seconds.
   See Profinet 2.4 Services, section 6.3.11.2.2 */
#define PF_DCP_SAM_TIMEOUT 3000000 /* microseconds */
```